app/mailers/user_mailer.rb

```diff
@@ -1,7 +1,14 @@
 class UserMailer < ApplicationMailer
+
   def reminder(urgent_items)
   	@urgent_items = urgent_items
   	mail(to: 'monikaglier@gmail.com', subject: 'Reminder from Todo List')
   end
+
+  def registration_confirmation(user)
+    @user = user
+    mail(to: 'monikaglier@gmail.com', subject: "Registration Confirmation")
```

Comment:
Teraz kiedy user ma własny email to wydaję mi się że każdy powinien otrzymywać własne emaile a nie Ty wszystkie ;) To samo się tyczy innych akcji w tym mailerze :)